src/DynamoCoreWpf/UI/GuidedTour/Guide.cs

```diff
@@ -209,6 +209,7 @@ internal void NextStep(int CurrentStepSequence)
             {
                 CalculateStep(GuideFlow.FORWARD, CurrentStepSequence);
                 CurrentStep.Show(GuideFlow.FORWARD);
+                Logging.Analytics.TrackEvent(Logging.Actions.Next, Logging.Categories.GuidedTourOperations, GuideNameResource, CurrentStep.Sequence);
```

Comment:
although the resource name is relatively more static, currently they are a bit misleading. e.g. The GettingStarted ones are for the first tour while the onboarding ones are for second tour, while the second tour itself is called Getting Started. If we go this way, I suggest we make a sweep update of all resource name to at least match the current tour names. Otherwise, we might get confused about our analytics data later

Reply:
On the other hand, we could use `Res.ResourceManager.GetString(nextGuide.GuideNameResource).Replace("_", "");`

Reply:
updated